setup.py

```diff
@@ -256,22 +257,15 @@ class OTIO_build_py(setuptools.command.build_py.build_py):
     """Stamps PROJECT_METADATA into __init__ files."""
 
     def run(self):
-        setuptools.command.build_py.build_py.run(self)
+        super().run()
 
-        if not self.dry_run:
+        if not self.dry_run and not self.editable_mode:
```

Comment:
`self.editable_mode` will be set to True when `-e` is given to the `pip install` command. See https://setuptools.pypa.io/en/latest/userguide/extension.html#setuptools.command.build.SubCommand.editable_mode.